### ISSUE TITLE
Upload Image Example not working with v6.4

### ISSUE THREAD

**Issue by tomway** [petersimonsson/libqatemcontrol#20]
After some troubleshooting, it appears that the qatemuploader example program never executes the "onFTDC" function.

**Comment by PNixx** [comment]
Doesn't work with v6.2, v6.3, v6.4, v6.6:
```
$ ./qatemuploader 192.168.1.240 1 ~/Desktop/1.png
Uploading...
```
And nothing... but wireshark all packages sent to ATEM. Exit with ctrl+c

**Comment by petersimonsson** [comment]
Hi,
Yes, I have screwed up the communication somewhere in there and I have had a hard time tracking down the problem. I plan to look at it again soon, so hopefully I manage to find the problem this time.

**Comment by petersimonsson** [comment]
I have fixed half of the problems for v6.6. The upload itself works now but getting the available video modes are broken atm. I haven't tested with older versions so it might work correctly on one of them.

**Comment by petersimonsson** [comment]
Added a fix for the available video modes as well... closing this ticket.
